clang/test/CAS/mccas-replay-test.cpp

```diff
@@ -0,0 +1,19 @@
+// RUN: rm -rf %t && mkdir -p %t
+// RUN: export LLVM_CACHE_CAS_PATH=%t/cas && %clang-cache \
+// RUN:   %clang -target x86_64-apple-macos11 -c -Xclang -fcas-backend -Rcompile-job-cache %s -o %t/tmp.o 2>&1 | FileCheck %s -check-prefix=CACHE-MISS
+// CACHE-MISS: remark: compile job cache miss
+
+// RUN: llvm-objdump -h %t/tmp.o 2>&1 | FileCheck %s -check-prefix=CACHE-MISS-FILE
+// CACHE-MISS-FILE-NOT: llvm-objdump: error: {{.*}}: The file was not recognized as a valid object file
```

Comment:
I would rather check correct header content, rather than rule out a specific error pattern. Imagine if the error message changed first, then this test is broken, this would not trigger any error.